cookbook/security/custom_authentication_provider.rst

```diff
@@ -127,7 +127,7 @@ set an authenticated token in the token storage if successful.
 
         public function __construct(TokenStorageInterface $tokenStorage, AuthenticationManagerInterface $authenticationManager)
         {
-            $this->tokenStorage = $tokenStorage;
+            $this->tokenStorage          = $tokenStorage;
```

Comment:
Sorry. I looked at the other constructor on this page where you do align it. I'll revert it. Thank you.